handsontable.d.ts

```diff
@@ -1577,6 +1578,14 @@ declare namespace Handsontable {
     unmodifyRow?: (row: number) => void;
   }
 
+  interface I18n {
+    dictionaryKeys: object;
+    registerLanguageDictionary: (languageCodeOrDictionary: any, dictionary: Object) => object;
+    getTranslatedPhrase: (dictionaryKey: string, extraArguments?: any) => object;
+    getLanguagesDictionaries: () => any[];
+    getLanguageDictionary: (languageCode: string) => object;
+  }
```

Comment:
I don't see this in the docs. Is it new?

If yes, it would probably also be a good idea to add a quick test to make sure there's no accidental regression in the future.

Reply:
I don't know if it's new, but I found it [here](https://docs.handsontable.com/pro/1.15.0/tutorial-internationalization.html). Tests added exept for `getTranslatedPhrase()` which seems to expect being called from a `Handsontable` instance. I'll try to figure out a way to get the phrase from an instance.